fs_extra_1.rmSync is not a function

### Current Behavior

Looks like rmSync is not found when called upon during the build process

`npm run build <app_name>`

```log
| => npm run build <app_name>

> project@0.0.0 build ...
> nx build "<app_name>"


> nx run app-name:build


  Building project  app-name ...

  Copying project files to a temporary folder
  Resolving dependencies...
The lock file does not exist. Locking.
The currently activated Python version 3.10.9 is not supported by the project (>=3.8,<3.10).
Trying to find and use a compatible version. 
Using python3.9 (3.9.16)
Creating virtualenv app-name in /app-name/.venv
Updating dependencies
Resolving dependencies... (0.1s)

Writing lock file


   ERROR  (0 , fs_extra_1.rmSync) is not a function


 —————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Ran target build for project app-name (1s)
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]
 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! project@0.0.0 build: `nx build "app-name"`
npm ERR! Exit status 1

```



Build command:

```json
"build": {
      "executor": "@nxlv/python:build",
      "outputs": ["apps/app-name/dist"],
      "options": {
        "outputPath": "apps/app-name/dist",
        "publish": true,
        "lockedVersions": true,
        "bundleLocalDependencies": true
      }
    },
```


node_modules/fs-extra/lib/remove/index.js

```js
'use strict'

const fs = require('graceful-fs')
const u = require('universalify').fromCallback

function remove (path, callback) {
  fs.rm(path, { recursive: true, force: true }, callback)
}

function removeSync (path) {
  fs.rmSync(path, { recursive: true, force: true })
}

module.exports = {
  remove: u(remove),
  removeSync
}

```


package.json
```json
...
  "devDependencies": {
    "@nrwl/cli": "14.8.5",
    "@nrwl/devkit": "^15.7.2",
    "@nrwl/workspace": "14.8.5",
    "@nxlv/python": "^15.5.0",
    "nx": "14.8.5",
    "prettier": "^2.6.2",
    "typescript": "~4.8.2"
  },
  "dependencies": {
    "fs-extra": "^11.1.0"
  }
```

```log
| => poetry --version
Poetry (version 1.3.2)
```

### Expected Behavior

npm run build app-name for a python app should build correctly. 

### Steps to Reproduce

1. npx  create-nx-workspace project-name --empty
2. npm install @nxlv/python --save-dev
3. curl -sSL https://install.python-poetry.org | python3 -
4. nx generate @nxlv/python:project app-name
5. npm run build app-name


### Nx Report

```shell
| => nx report

 >  NX   Report complete - copy this into the issue template

   Node : 19.6.0
   OS   : darwin x64
   npm  : 9.4.0
   
   nx : 14.8.5
   @nrwl/angular : Not Found
   @nrwl/cypress : Not Found
   @nrwl/detox : Not Found
   @nrwl/devkit : 15.7.2
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : Not Found
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 14.8.5
   @nrwl/js : Not Found
   @nrwl/linter : 14.8.5
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/rollup : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : Not Found
   @nrwl/web : Not Found
   @nrwl/webpack : Not Found
   @nrwl/workspace : 14.8.5
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
   	 @nxlv/python: 15.5.0
```


### Failure Logs

_No response_

### Additional Information

_No response_

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     "@swc/cli": "~0.1.61",
     "@swc/core": "^1.3.26",
     "@types/cross-spawn": "^6.0.2",
+    "@types/fs-extra": "^11.0.1",
     "@types/jest": "28.1.1",
     "@types/node": "18.13.0",
     "@typescript-eslint/eslint-plugin": "^5.52.0",
@@ -57,6 +58,7 @@
     "command-exists": "^1.2.9",
     "cross-spawn": "^7.0.3",
     "file-uri-to-path": "^2.0.0",
+    "fs-extra": "^11.1.0",
     "tslib": "^2.3.0",
     "uuid": "^9.0.0"
   },

--- a/packages/nx-python/src/executors/build/executor.ts
+++ b/packages/nx-python/src/executors/build/executor.ts
@@ -6,7 +6,7 @@ import {
   readFileSync,
   writeFileSync,
   mkdirSync,
-  rmSync,
+  removeSync,
 } from 'fs-extra';
 import { join } from 'path';
 import {
@@ -60,9 +60,7 @@ export default async function executor(
       if (!options.ignorePaths.includes(file)) {
         const source = join(root, file);
         const target = join(buildFolderPath, file);
-        copySync(source, target, {
-          recursive: true,
-        });
+        copySync(source, target);
       }
     });
 
@@ -102,13 +100,13 @@ export default async function executor(
     writeFileSync(buildPyProjectToml, stringify(buildTomlData));
     const distFolder = join(buildFolderPath, 'dist');
 
-    rmSync(distFolder, { recursive: true, force: true });
+    removeSync(distFolder);
 
     logger.info(chalk`  Generating sdist and wheel artifacts`);
     const buildArgs = ['build'];
     runPoetry(buildArgs, { cwd: buildFolderPath });
 
-    rmSync(options.outputPath, { recursive: true, force: true });
+    removeSync(options.outputPath);
     mkdirSync(options.outputPath, { recursive: true });
     logger.info(
       chalk`  Artifacts generated at {bold ${options.outputPath}} folder`
@@ -116,7 +114,7 @@ export default async function executor(
     copySync(distFolder, options.outputPath);
 
     if (!options.keepBuildFolder) {
-      rmSync(buildFolderPath, { recursive: true, force: true });
+      removeSync(buildFolderPath);
     }
 
     return {

--- a/packages/nx-python/src/executors/build/resolvers/utils.ts
+++ b/packages/nx-python/src/executors/build/resolvers/utils.ts
@@ -12,7 +12,7 @@ export function includeDependencyPackage(
     const pkgFolder = join(root, pkg.include);
     const buildPackageFolder = join(buildFolderPath, pkg.include);
 
-    copySync(pkgFolder, buildPackageFolder, { recursive: true });
+    copySync(pkgFolder, buildPackageFolder);
 
     buildTomlData.tool.poetry.packages.push({
       include: pkg.include,

--- a/packages/nx-python/src/executors/sls-deploy/executor.ts
+++ b/packages/nx-python/src/executors/sls-deploy/executor.ts
@@ -4,7 +4,7 @@ import spawn from 'cross-spawn';
 import { Logger } from '../utils/logger';
 import { ExecutorSchema } from './schema';
 import path from 'path';
-import { existsSync, readdirSync, writeFileSync, rmSync } from 'fs-extra';
+import { existsSync, readdirSync, writeFileSync, removeSync } from 'fs-extra';
 
 const logger = new Logger();
 
@@ -61,6 +61,6 @@ export default async function executor(
       success: false,
     };
   } finally {
-    rmSync(requirementsTxt, { recursive: true, force: true });
+    removeSync(requirementsTxt);
   }
 }

--- a/packages/nx-python/src/executors/sls-package/executor.ts
+++ b/packages/nx-python/src/executors/sls-package/executor.ts
@@ -4,7 +4,7 @@ import spawn from 'cross-spawn';
 import { Logger } from '../utils/logger';
 import { ExecutorSchema } from './schema';
 import path from 'path';
-import { existsSync, readdirSync, writeFileSync, rmSync } from 'fs-extra';
+import { existsSync, readdirSync, writeFileSync, removeSync } from 'fs-extra';
 
 const logger = new Logger();
 
@@ -59,6 +59,6 @@ export default async function executor(
       success: false,
     };
   } finally {
-    rmSync(requirementsTxt, { recursive: true, force: true });
+    removeSync(requirementsTxt);
   }
 }

